assemble/bin/accumulo-cluster

```diff
@@ -59,16 +59,16 @@ function verify_config {
 
   unset manager1
   if [[ -f "${conf}/$manager_file" ]]; then
-    manager1=$(egrep -v '(^#|^\s*$)' "${conf}/$manager_file" | head -1)
+    manager1=$(grep -E -v '(^#|^\s*$)' "${conf}/$manager_file" | head -1)
```

Comment:
Single-character options can be combined. This doesn't really matter, but the end result of converting from `egrep -v` to `grep -Ev` is that it is the same number of characters, rather than `grep -E -v`, which is 2 more.

```suggestion
    manager1=$(grep -Ev '(^#|^\s*$)' "${conf}/$manager_file" | head -1)
```